### PR TITLE
console message improvement for VersionReleaser

### PR DESCRIPTION
**Pull request by deki**
use resource bundle instead of string constant, also print a message on success)

### CODE REVIEW

File: src/test/java/hudson/plugins/jira/JiraReleaseVersionUpdaterTest.java [hunk 1]
```diff
@@ -47,6 +48,7 @@ public void createMocks() {
 		env = mock(EnvVars.class);
 		project = mock(AbstractProject.class);
 		site = mock(JiraSite.class);
+      logger = mock(PrintStream.class);
```
Comment (warden) [inline src/test/java/hudson/plugins/jira/JiraReleaseVersionUpdaterTest.java]: Please fix indentation in the test class

**Comment by deki** [comment]
tabs and spaces are mixed in several classes, what is preferred?

**Comment by warden** [comment]
Please use spaces as the preferred way, thanks!

**Comment by deki** [comment]
Ok I already used spaces in my commit, so I will replace the tabs in the other lines.

**Pull request merged**